cmd/gateway/main.go

```diff
@@ -1,6 +1,7 @@
 package main
 
 import (
+	"fmt"
 	"log"
```

Comment:
Do you still need log?

Reply:
It's still used to `log.Fatal` before the zap logger is configured [e.g.](https://github.com/alexdunne/gs-onboarding/blob/623ea5cb10cf3a2f7aad2e2b30fcc7083b7d6efc/cmd/gateway/main.go#L36)